index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!-- <h2>Hello World</h2> -->
+
+    <main class="main-wrapper">
+      <div class="header">
+        <h1 id="title">
+          Survey Form
+        </h1>
+        <p id="description">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Voluptate quasi molestias sunt, ullam placeat qui commodi libero porro dignissimos blanditiis et animi doloremque rem, ipsam numquam omnis deserunt, quos iste.
+        </p>
+      </div>
+
+      <section class="form-section">
+        <form action="" id="survey-form">
+          <div class="form__group">
+            <input type="text" class="form__group-input" id="name" placeholder="Name">
+            <label for="name" id="name-label" class="form__group-label">Name</label>
+          </div>
+  
+          <div class="form__group">
+            <input type="email" class="form__group-input" id="email" placeholder="Email">
+            <label for="email" id="email-label" class="form__group-label">Email</label>
+          </div>
+  
+  
+          <div class="form__group">
+            <input type="number" class="form__group-input" id="number" placeholder="Number" min="2" max="50">
+            <label for="number" id="number-label" class="form__group-label">Number</label>
+          </div>
+  
+          <div class="form__group">
+            <select class="form__group-select" name="dropdown" id="dropdown">
+              <option value="Male">Male</option>
+              <option value="Female">Female</option>
+              <option value="Other">Other</option>
+            </select>
+          </div>
+  
+          <div class="form-radio">
+            <div class="form__group-radio__button">
+              <input type="radio" class="form__group-input" id="radio-button" name="radio-button">
+              <label for="radio-button">Option 1</label>
+            </div>
+  
+            <div class="form__group-radio__button">
+              <input type="radio" class="form__group-input" id="radio-button" name="radio-button">
+              <label for="radio-button">Option 2</label>
+            </div>
+  
+            <div class="form__group-radio__button">
+              <input type="radio" class="form__group-input" id="radio-button" name="radio-button">
+              <label for="radio-button">Option 3</label>
+            </div>
+          </div>
```

Comment:
The `id` attribute should be unique in your html document. Please remove duplicate IDs ''radio-button"